test/Elastic.Apm.MongoDb.Tests/LICENSE.txt

```diff
@@ -0,0 +1,2 @@
+The code in this directory is based on the elastic-apm-mongo project by Vadim Hatsura (@vhatsura),
```

Comment:
I think this should be concatenated with the contents of LICENSE. An MSBuild target might be a good approach to handle this.

Reply:
I agree. This would improve clarity around which terms apply to which source files!

Reply:
I think the comment below covers this topic, let's this discuss it in https://github.com/elastic/apm-agent-dotnet/pull/1215#discussion_r609254103